Sources/DateScrubber.swift

```diff
@@ -69,20 +81,48 @@ public class DateScrubber: UIViewController {
         }
     }
 
+    private var sectionLabelState = State.Inactive {
+        didSet {
+
+            if self.sectionLabelState == .Active {self.setSectionLabelActive()}
+            if self.sectionLabelState == .Inactive {self.setSectionLabelInactive()}
+            self.updateSectionLabelFrame()
+        }
+    }
+
+    private var scrubberState = State.Inactive {
+        didSet {
+            self.updateDateScrubberFrame()
+        }
+    }
+
```

Comment:
So it handles visibility, we could probably use a name that better reflects that. 🤔 I'm not that good with naming either. 😰 Thinking about it.